tree/modules/packages/apt_get

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/python
+
```

Comment:
how will it be distributed to the nodes ? I don't see anything related to copy/deploiment of this